spynnaker_external_devices_plugin/pyNN/control_models/munich_motor_control.py

```
@@ -31,6 +31,7 @@ class MunichMotorControl(AbstractPopulationVertex):
     _N_ATOMS = 6
 
     CORE_APP_IDENTIFIER = constants.MUNICH_MOTOR_CONTROL_CORE_APPLICATION_ID
+    _model_based_max_atoms_per_core = 6
```

Should maybe throw an exception when n_neurons is not 6